docs/ref/models/database-functions.txt

```diff
@@ -716,6 +738,25 @@ Usage example::
     >>> print(author.screen_name)
     Margaret Smith (Maggie)
 
+``Left``
+--------
+
+.. class:: Left(expression, length, **extra)
+
+.. versionadded:: 2.1
+
+Accepts a single text field or expression and the 1-indexed length of the
+substring to return. The first ``length`` characters are returned. If length
```

Comment:
I'm making a few small edits to this PR and don't understand "If length is negative, ``None`` is returned." Is that leftover from an earlier implementation -- now that length is validated to be greater than zero? Or have I missed the meaning?

Reply:
Oh, oops.  Yes, that was in a previous commit.